kinetic/distribution.yaml

```
@@ -11593,6 +11593,11 @@ repositories:
       url: https://github.com/ros-controls/ros_controllers.git
       version: kinetic-devel
     status: maintained
+  ros_cvb_camera_driver:
+    doc:
+      type: git
+      url: https://github.com/gleichaufjo/ros_cvb_camera_driver.git
+      version: kinetic-devel
```

There's no `kinetic-devel` branch only master. If we're going to index that it would need to be as follows.

```suggestion
      version: master
```